src/shop/ordering/orders.js

```diff
@@ -86,6 +97,22 @@ bloombox.shop.order.Type = {
 };
 
 
+/**
+ * Specifies statuses an order may be in.
+ *
+ * @enum {number}
+ * @export
+ */
+bloombox.shop.order.Status = {
+  'PENDING': proto.opencannabis.commerce.OrderStatus.PENDING,
+  'REJECTED': proto.opencannabis.commerce.OrderStatus.REJECTED,
```

Comment:
frontend-facing enum for order status is now available at `bloombox.shop.order.Status`